test/arkane/arkaneKineticsTest.py

```diff
@@ -52,48 +49,44 @@ def test_give_tlist_for_kineticsjob(self):
         """
         rxn = Reaction(transition_state=TransitionState())
         t_list = [50.7, 100, 300, 800, 1255]
-        kjob = KineticsJob(rxn, Tlist=(t_list, 'K'))
-        self.assertEqual(min(t_list), kjob.Tmin.value_si)
-        self.assertEqual(max(t_list), kjob.Tmax.value_si)
-        self.assertEqual(len(t_list), kjob.Tcount)
+        kjob = KineticsJob(rxn, Tlist=(t_list, "K"))
+        assert min(t_list) == kjob.Tmin.value_si
+        assert max(t_list) == kjob.Tmax.value_si
+        assert len(t_list) == kjob.Tcount
 
     def test_give_temperature_range_for_kineticsjob(self):
         """
         Ensures that Tlist is set when a range of temperatures is specified
         """
         rxn = Reaction(transition_state=TransitionState())
-        kjob = KineticsJob(rxn, Tmin=(50, 'K'), Tmax=(4000, 'K'), Tcount=5)
-        self.assertEqual(5, len(kjob.Tlist.value_si))
-        self.assertEqual(50, min(kjob.Tlist.value_si))
-        self.assertAlmostEqual(4000, max(kjob.Tlist.value_si))
+        kjob = KineticsJob(rxn, Tmin=(50, "K"), Tmax=(4000, "K"), Tcount=5)
+        assert 5 == len(kjob.Tlist.value_si)
+        assert 50 == min(kjob.Tlist.value_si)
+        assert round(abs(4000 - max(kjob.Tlist.value_si)), 7) == 0
         inverse_tlist = 1 / kjob.Tlist.value_si
-        self.assertAlmostEqual(inverse_tlist[1] - inverse_tlist[0],
-                               inverse_tlist[-1] - inverse_tlist[-2],
-                               msg='The points for fitting do not appear 1/T spaced. '
-                                   'Obtained values of {0} and {1}'.format(inverse_tlist[1] - inverse_tlist[0],
-                                                                           inverse_tlist[-1] - inverse_tlist[-2]))
+        assert (
+            round(abs(inverse_tlist[1] - inverse_tlist[0] - inverse_tlist[-1] - inverse_tlist[-2]), 7) == 0
```

Comment:
hang on, shouldn't _A-B≈C-D_  become _(A-B)-(C-D)≈0_ but this is _(A-B-C-D)≈0_  ?  And how many more of these do we need to check? Is there a systematic way to check them all?

Reply:
Ah, thanks for catching that. The original conversion did this in a few places and I fixed _almost_ all of them, apparently. Latest commit (46a8b8b) should have caught the last of them.

I find them using the global search in vscode with regular expressions, finding `round(abs(` where the parentheses then contain more than one `-` symbol.